v-next/hardhat-chai-matchers/test/fixture-projects/hardhat-project/artifacts/contracts/ChangeEtherBalance.sol/ChangeEtherBalance.dbg.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
TMP solution: parallel compilation is currently not supported (missing mutex and no cache). So I temporarily copied the already compiled contracts